modules/analysis/src/endpoints/tests/mod.rs

```diff
@@ -398,8 +398,8 @@ async fn find_component_by_query(ctx: &TrustifyContext) -> Result<(), anyhow::Er
     };
 
     for each in [
```

Comment:
@ctron for these tests to pass, the ones including a `purl` field require comparisons against a stringified representation of a PURL, which used to come from the `get_purl` proc but now come from `sbom_package_purl_ref.purl`.

Reply:
@ctron here